### ISSUE TITLE
possible memory leak

### ISSUE THREAD

**Issue by LianaHus** [ethereum/remix-ide#2893]
Firefox is hanging when autocompile is on and one edit the text in editor multiple times in a row a bit fast.
Debugger is causing same also in chrome

**Comment by AlphaSerpentis** [comment]
I've temporarily solved this by turning auto-compile off, but after compilation, it'll hang for some reason then resume normally the best case. The worst case is that it will hang and never recover while still trying to read/write data. This is especially noticeable with a file that contains hundreds of lines (700 lines for me).

From what I could observe, Remix is writing/reading data for 30 seconds+ at speeds of excess of 120 MB/s.

**Comment by VexyCats** [comment]
Yes the same issue is happening on chrome. So bad that it cannot be used without having auto compile turned off.
